conf/auth.conf

```diff
@@ -57,6 +57,27 @@
 # (ie exactly as if auth yes was present).
 #
 
+# CONTROLLING FILE ACCESS (previously in fileserver.conf)
+
+# In previous versions of Puppet, you controlled file access by adding
+# rules to fileserver.conf. In Puppet 5 with Puppet Server, you can control
+# file access in auth.conf by controlling the /file_metadata(s)/<mount point>
+# and /file_content(s)/<mount point> paths. See the Puppet Server
+# documentation at
+# https://docs.puppet.com/puppetserver/latest/config_file_auth.html.
```

Comment:
@joshcooper Would files that go through the `static_file_content` be able to reference fileserver mount points?  I'd been thinking that for the typical use case where the `versioned-code` service is configured with Puppet Server that the `code-content-command` it would run for a `static_file_content` request would be a script which in many cases wouldn't know anything about fileserver mount points - e.g., just does a `git show <version>|<filename>`.